frontend/tests/features/collection/collection.test.ts

```diff
@@ -67,76 +73,94 @@ describe("Collection", () => {
 
     describe("Publish a collection", () => {
       describe("when no dataset", () => {
-        it("shows disabled publish button", async () => {
-          await login();
+        test("shows disabled publish button", async ({ page }) => {
+          await login(page);
 
-          await createCollection();
+          await createCollection({ page });
 
-          await tryUntil(async () => {
-            const publishButton = await page.$(
-              getTestID("publish-collection-button")
-            );
+          await tryUntil(
+            async () => {
+              const publishButton = await page.$(
+                getTestID("publish-collection-button")
+              );
 
-            expect(await publishButton?.getAttribute("disabled")).toBe("");
-          }, 100);
+              expect(await publishButton?.getAttribute("disabled")).toBe("");
+            },
+            { maxRetry: 100, page }
+          );
         });
       });
     });
   });
 
   describeIfDevStaging("Deployed Env Tests", () => {
     describe("invalid DOIs", () => {
-      it("doesn't create a collection with a DOI in an invalid format", async () => {
+      test("doesn't create a collection with a DOI in an invalid format", async ({
+        page,
+      }) => {
         const timestamp = Date.now();
 
-        await login();
-        await showCreateForm();
-        await populateRequiredInputs({
-          ...TEST_COLLECTION,
-          name: "TEST_COLLECTION" + timestamp,
-        });
+        await login(page);
+        await showCreateForm(page);
+        await populateRequiredInputs(
+          {
+            ...TEST_COLLECTION,
+            name: "TEST_COLLECTION" + timestamp,
+          },
+          page
+        );
 
         // Specify a DOI that is in an invalid format.
-        await populatePublicationDOI("10.1016/j.2022.104097");
+        await populatePublicationDOI("INVALID_FORMAT", page);
```

Comment:
@maniarathi caught this typo earlier this year, I think it's time to correct it!

Essentially the comment at 113 says we should be testing for invalid format, but in fact we're testing a valid format but non-existent DOI here, whereas the test at L138 is testing opposite